src/mlpack/methods/random_forest/random_forest.hpp

```diff
@@ -179,7 +181,8 @@ class RandomForest
                const double minimumGainSplit = 1e-7,
                const size_t maximumDepth = 0,
                DimensionSelectionType dimensionSelector =
-                   DimensionSelectionType());
+                   DimensionSelectionType(),
+               bool warmStart = false);
```

Comment:
This is pedantic, but we can make it `const bool warmStart`.  Also, I might suggest moving this option before `dimensionSelector`.  The reason I say that is that I think it is more common to use `warmStart`, and also since our next release is mlpack 4, we can break reverse compatibility.  (And honestly I think this would break very few people's code.  Specifying an instantiated `dimensionSelector` object is a niche usage in my opinion.)